test/blackbox-tests/test-cases/pkg/opam-var/dune

```diff
@@ -4,12 +4,19 @@
 
 (data_only_dirs fake_opam_root)
 
+; Without this, the .opam-switch directory won't be copied to the build directory.
+
+(subdir
+ fake_opam_root/default
+ (dirs :standard .opam-switch))
```

Comment:
Erm. That's strange. We mark `fake_opam_root` as a data_only_dirs. Shouldn't everything inside be copied? Seems like a bug @rgrinberg.

Reply:
The comment should read:

```
;; Without this, no copying rules will be setup for the .opam-switch directory
```

Things are actually copied when there are dependencies on them.